site/docs/pages/identity/get-avatar.mdx

```diff
@@ -8,14 +8,20 @@ use it with Next.js or any Node.js backend.
 
 Supported providers:
 
+- Basenames
 - ENS
 
 ## Usage
 
 ```tsx
 import { getAvatar } from '@coinbase/onchainkit/identity';
+import { base, mainnet } from 'viem/chains';
 
-const avatar = await getAvatar({ ensName: 'vitalik.eth' });
+// For Basename
+const baseAvatar = await getAvatar({ ensName: 'vitalik.base.eth', chain: base });
+
+// For ENS
+const ensAvatar = await getAvatar({ ensName: 'vitalik.eth', chain: mainnet });
```

Comment:
Let's split those example in two code block, and also add Return Value tab

Reply:
This is what the return value tab would look like:

baseEnsAvatar: https://zku9gdedgba48lmr.public.blob.vercel-storage.com/basenames/avatar/paul.base.eth/1722120524815/FWUzoZmJ_400x400-kWjr2gMvjNe9hHMs9Z9LxGVGIME3By.jpg

mainnetEnsAvatar https://euc.li/paulcramer.eth

Reply:
done